server/src/github.com/alexeyknyshev/cpsrv/cpsrv.go

```diff
@@ -39,54 +38,36 @@ type Message struct {
 	Text string `json:"text"`
 }
 
-func checkConvertionUint(val uint32, err error, context string) uint32 {
-	if err != nil {
-		log.Printf("%s: uint conversion err => %v\n", context, err)
-		return 0
-	}
-	return val
+type HandlerContextStruct struct {
+	Tnt *tarantool.Connection
+	Logger
```

Comment:
Why don't store logger as pointer (as you do with tnt) here and fill it with new value while creating new HandlerContext?